.drone.yml

```diff
@@ -91,6 +91,17 @@ pipeline:
     when:
       event: tag
 
+  trigger-e2e-tests:
+    image: quay.io/ukhomeofficedigital/drone-trigger:latest
```

Comment:
Should we fix at a version of drone-trigger so it doesn't break if the project gets an update?